### PR TITLE
brainworkshop: init at 2020-03-10

### PR DESCRIPTION
**Pull request by mt-caret**
###### Motivation for this change

Adds brainworkshop, a Dual-n-back training application. Depends on https://github.com/NixOS/nixpkgs/pull/82219.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/games/brainworkshop/default.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{ stdenv, lib, fetchFromGitHub, python36 }:
```
Comment (symphorien) [inline pkgs/games/brainworkshop/default.nix]: Is there a reason to hardcode `python36` instead of just `python3` ?

**Comment by symphorien** [comment]
the program starts correctly, but I can't hear any sound.

File: pkgs/games/brainworkshop/default.nix (hunk 1 above)
Comment (mt-caret) [reply pkgs/games/brainworkshop/default.nix]: This is a workaround as python3 is 3.8 something, which breaks pyglet (https://github.com/pvcraven/arcade/issues/514)

**Comment by mt-caret** [comment]
Yeah; I'm digging through the code but I can't find why that's the case. The game is unplayable in its current state, and since I don't really have time to fix this right now, I guess this PR should be closed.

**Pull request closed without merging**